inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/http/HttpBaseSource.java

```diff
@@ -77,7 +78,12 @@ public synchronized void start() {
                     statIntervalSec, maxMonitorCnt);
         }
         // register metrics
-        this.metricItemSet = new DataProxyMetricItemSet(this.getName());
+        ConfigManager configManager = ConfigManager.getInstance();
+        String clusterId =
+                configManager.getCommonProperties().getOrDefault(
+                        ConfigConstants.PROXY_CLUSTER_NAME, "DataProxy");
```

Comment:
Suggest adding a constant for the default cluster value, such as `DEFAULT_PROXY_CLUSTER_NAME`.